scripts/mkcloud

```diff
@@ -1221,6 +1222,12 @@ function runupdate
     return $?
 }
 
+function runupdate_clients
```

Comment:
declaring this as a function is not enough to make it a step, it is not yet known by the step validator